docs/scale/03-create-cluster.md

```diff
@@ -0,0 +1,332 @@
+        12fd2a115d85        bridge              bridge   
+
+3.  Now, run the same command to get cluster information.
+
+    Provided you set `export DOCKER_HOST="tcp://192.168.33.11:3375"``, the
+    command directs to the Swarm port and returns information from each node in
+    the cluster.
+
+          $ docker network ls
+          NETWORK ID          NAME                DRIVER
+          82ce2adce6a7        store/bridge        bridge              
+          c3ca43d2622d        store/host          host                
+          d126c4b1e092        frontend01/host     host                
+          6ea89a1a5b6a        frontend01/bridge   bridge              
+          d3ddb830d7f5        frontend02/host     host                
+          44f238353c14        frontend02/bridge   bridge              
+          c500baec447e        frontend02/none     null                
+          77897352f138        store/none          null                
+          8060bd575770        frontend01/none     null                
+          429e4b8c2c8d        worker01/bridge     bridge              
+          57c5001aca83        worker01/none       null                
+          2101103b8494        worker01/host       host        
+
+4. Create the overlay network with the `docker network` command
+
+        $ docker network create --driver overlay mynet
+
+5. Repeat the two network commands again to see how the network list has changed.   
+
+        docker network ls
+        docker -H=tcp://192.168.33.11:2375 network ls
+
+    As all Swarm nodes in your environment are configured to use the Consul
+    discovery service at `consul://192.168.33.11:8500`, they all should see the
+    new overlay network. Verify this with the next step.
+
+3. Try running a network command on an individual node, for example to run it on
+the `frontend01` node:
+
+          docker -H=tcp://192.168.33.20:2375 network ls
+
+    You should see an entry for the `mynet` network using the `overlay` driver
+    as shown above. You would get the same output if your ran the command from
+    node's command line.
+
+## Step 4: Review your work
+
+The diagram below shows the complete cluster configuration including the overlay
+container network, `mynet`. The `mynet` is shown as red and is available to all
+Docker hosts using the Consul discovery backend. Later in the procedure you will
+connect containers to this network.
+
+![](../images/overlay-review.jpg)
+
+The `swarm` and `consul` containers on the `manager` node are not attached to
+the `mynet` overlay network. These containers are running on `manager` nodes
+default bridge network. To verify this try using these two commands:
+
+      docker -H=tcp://192.168.33.11:2375 network inspect bridge
+      docker -H=tcp://192.168.33.11:2375 network inspect mynet
+
+You should find two containers running on the `manager` node's `bridge` network
+but nothing yet on the `mynet` network.
```

Comment:
In `overlay-review.jpg` `mynet` is associated with `manager`. I think it should be clarified.